deployment/hasura/metadata/actions.graphql

```diff
@@ -34,29 +34,29 @@ type Mutation{
 
 type Query {
   getModelEffectiveArguments(
-    missionModelId: ID!
+    missionModelId: Int!
```

Comment:
`getModelEffectiveArguments` and `getActivityEffectiveArguments` are handled in `merlin-server`. I don't think we should change them as part of this PR.

Just a brief survey of some of the files in `merlin-server` that will need to change:
* HasuraAction
* HasuraParsers
* MissionModelService interface
* LocalMissionModelService
* StubMissionModelService
* MissionModelRepository interface
* PostgresMissionModelRepository
* LocalMissionModelRepository

I think we absolutely should change them, but I think that is well out of scope for this PR.

Reply:
Yeah, I'm with you Matt. Appreciate the fix, Dylan, but it's probably best to keep this PRs scope in-check

Reply:
When we have API tests for more of our actions, we will hopefully get notified automatically about issues like this:
![Screen Shot 2022-03-31 at 5 54 44 PM](https://user-images.githubusercontent.com/1189602/161173711-e1f46ff1-3d83-424a-9cd4-6bfe4581c345.png)

Reply:
Latest push reverts the `ID` -> `Int` change for `missionModelId`